packages/theme-core-plugin--sass/sass.js

```diff
@@ -60,14 +98,17 @@ module.exports = (userConfig) => {
         });
   }
 
+
   function compile(done) {
+    if (config.lint.enabled) {
+      validateCssWithNoExit();
+    }
     cssCompile(done, true);
   }
```

Comment:
Doing things this way enforces a validate then compile... but it doesn't block compile with an exit on validate failing. I see why you're doing this, but part of this modular nature is to provide a lot of tasks and then let the users put them together how they'd like. I'd prefer to not have this.